src/boot_semver/core.clj

```diff
@@ -90,4 +93,9 @@
       (when (and (nil? (:no-update *opts*)) (not= curver version))
         (util/info (clojure.string/join ["Updating Project Version...: " curver "->" version "\n"]))
         (set-version! semver-file version))
-      fs)))
+      (if (:include *opts*)
+        (-> fs (boot/add-resource
```

Comment:
This is great, as I was doing the same thing in [`lambone`](https://github.com/Lambda-X/lambone/blob/master/resources/leiningen/new/lambone/common/dev/boot.clj#L41)